cachi2/core/models/input.py

```diff
@@ -73,6 +73,7 @@ class BundlerPackageInput(_PackageInputBase):
     """Accepted input for a bundler package."""
 
     type: Literal["bundler"]
+    allow_binary: bool = False
```

Comment:
I believe the CLI change should go to a separate commit and be decoupled from the core logic.